tinydtls-sys/src/lib.rs

```diff
@@ -433,8 +434,6 @@ mod tests {
         // this pointer should be valid up until here.
         unsafe {
             dtls_free_context(server_context);
-            // To ensure that we don't use the context pointer sometime later (in case the tests are extended later on), we drop the pointer.
-            std::mem::drop(server_context);
```

Comment:
This is probably simply not needed anymore in the latest version, right?

Reply:
Apparently, according to clippy, this never had any effect at all :sweat_smile: